src/api/project/project.type.ts

```diff
@@ -0,0 +1,7 @@
+export interface Project {
+  name: string;
+  description: string;
+  isDeleted: boolean;
+}
+
+export type ProjectRequest = Project;
```

Comment:
`ProjectRequest` ini harusnya cuma `name` sama `description` aja, `isDeleted` gak perlu. Untuk bikin kyk gitu bisa pake fungsi `Omit` nya typescript. 

```javascript
export type ProjectRequest = Omit<Project, "isDeleted">
```
ini penjelasannya :
https://www.typescriptlang.org/docs/handbook/utility-types.html#omittk

Terus `isDeleted` ini dibuat defaultnya false di backend setiap kali ada project yang di create. Setupnya bisa di modelnya : 
https://mongoosejs.com/docs/defaults.html#declaring-defaults-in-your-schema